editors/code/syntaxes/clarity.tmLanguage.json

```diff
@@ -156,10 +156,10 @@
     },
     "tuple-key": {
       "name": "entity.name.type.clarity",
-      "match": "(?x)\n  ([a-z][a-zA-Z0-9_-]+)(?=:)\n"
+      "match": "(?x)\n  ([a-z][a-zA-Z0-9_-]*)(?=:)\n"
     },
     "args": {
-      "begin": "(?x)\n  (?<=^|[(]) \\s*\n  ([a-z][a-zA-Z0-9_-]+) \\s*\n",
+      "begin": "(?x)\n  (?<=^|[(]) \\s*\n  ([a-z][a-zA-Z0-9_-]*) \\s*\n",
```

Comment:
I noticed that the `use-trait` keyword is not getting recognized, is this something we could fix here as well?

Reply:
Done - added `use-trait` at `define-var` level and right now it is highlighted correctly.